Bottom: ca0891a2a16862ef8306ab3362aed3554d274d68
Top:    55d8b50fe4410f4902248d7f2891115b9b088b83
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-03-22 17:24:18 +0800

Refresh of patch

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index fdd5bcc..f0fe228 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -539,7 +539,7 @@ static int sk_has_rx_data(struct sock *sk)
 static bool sk_rx_array_has_data(struct sock *sk)
 {
 	struct socket *sock = sk->sk_socket;
-	struct skb_array *skb_array = tap_get_skb_array(sock->file);
+	struct skb_array *skb_array = tun_get_skb_array(sock->file);
 
 	if (skb_array)
 		return !__skb_array_empty(skb_array);
@@ -550,7 +550,7 @@ static bool sk_rx_array_has_data(struct sock *sk)
 static int sk_rx_array_length(struct sock *sk)
 {
 	struct socket *sock = sk->sk_socket;
-	struct skb_array *skb_array = tap_get_skb_array(sock->file);
+	struct skb_array *skb_array = tun_get_skb_array(sock->file);
 
 	if (skb_array)
 		return skb_array_peek_queue_len(skb_array);
